Add optional anonymous Tempo usage reporting (#1481)
grafana/tempo: docs/tempo/website/configuration/_index.md

```

Additional search-related settings are available in the [distributor](#distributor) and [ingester](#ingester) sections.

> 

Reviewer: What might make even more sense it to provide an endpoint (`/status/usage-stats`) that just dumps all of the information included in usage-stats. Is this even possible?

I don't consider that a requirement for this PR. Can you open an issue to add the new status endpoint and then we can document it here once it exists?

> 

Author: Nice, I like that idea.  I'll file an issue.

> 

Author: https://github.com/grafana/tempo/issues/1569

## Usage-report

By default, Tempo will report anonymous usage data to Grafana Labs concerning
the shape of a deployment.  This data is used to determine how common
the deployment of certain features are, or if a feature flag has been enabled,
replication factor or compression levels, etc.

> The reporting is controlled by a configuration option.  To disable the
> automatic reporting of this generic information, the following configuration
> can be used.

Reviewer 2: ```suggestion
The reporting is controlled by a configuration option. You can disable the
automatic reporting of this generic information using the following configuration:
```

> By default, Tempo will report anonymous usage data to Grafana Labs concerning
> the shape of a deployment.  This data is used to determine how common
> the deployment of certain features are, or if a feature flag has been enabled,
> replication factor or compression levels, etc.
> 
> The reporting is controlled by a configuration option.  To disable the
> automatic reporting of this generic information, the following configuration
> can be used.

Reviewer 2: ```suggestion
By default, Tempo will report anonymous usage data about the shape of a deployment 
to Grafana Labs. This data is used to determine how common
the deployment of certain features are, or if a feature flag has been enabled,
replication factor or compression levels, etc.
 
Reporting is controlled by a configuration option. You can disable the
automatic reporting of this generic information using the following configuration:
```
